src/screens/Admin/screens/InstructorPulse/index.js

```diff
@@ -10,13 +12,15 @@ export default ({instructors}) => (
     main={
       <InstructorsList
         title={unpublishedTitleText}
-        instructors={filter(instructors, ['published_lessons', 0])}
+        instructors={filter(instructors, ['published_lessons', 0])
+          .sort(sortBy('-submitted_lessons', '-claimed_lessons', '-approved_lessons', '-id'))}
```

Comment:
these lines are where the magic happens. *[sort-by](https://www.npmjs.com/package/sort-by)* seemed reasonable for the task.

1. Are they waiting for approval?
2. Have they claimed a lessons?
3. Do they have an approved lesson?
4. Newest first.